### ISSUE TITLE
PyCapsule_Import could not import module "datetime"

### ISSUE THREAD

**Issue by jlroles** [numpy/numpy#14931]
[SarimaForecasting.zip](https://github.com/numpy/numpy/files/3860457/SarimaForecasting.zip)
Python 3.7 installed from Anaconda
Error was received when running from Anaconda prompt.  Have received the same error when running programs from Spyder.  The problem was resolved when I reset Spyder.  This program won't run the function in parallel unless it is run from the Anaconda prompt.
Jeff Roles, PhD
jroles@hesperosinc.com

**Comment by mattip** [comment]
Possible duplicate of gh-14474

**Comment by mattip** [comment]
Does your machine also have a large number of cores? If so, can you try using  [`threadpoolctl`](https://pypi.org/project/threadpoolctl/) to manage the number of threads each process opens

**Comment by jlroles** [comment]
My machine has 8 cores.  I tried using threadpoolctl and I also tried running without the parallel option-both resulted in the same error.
